server/graphql/stocktake_line/src/mutations/update.rs

```diff
@@ -13,7 +14,9 @@ use service::{
     },
 };
 
-use super::{AdjustmentReasonNotProvided, StockLineReducedBelowZero};
+use crate::mutations::AdjustmentReasonNotValid;
```

Comment:
This was previously standard error, with no way to point to an erroneous line, change to structured.